src/pages/manageUser.tsx

```diff
@@ -0,0 +1,200 @@
+import { useTheme } from '@mui/material/styles';
+import { useTranslation } from 'next-i18next';
+
+const manageUser = () => {
+  const { t } = useTranslation();
+  const theme = useTheme<any>();
+  const router = useRouter();
+
+  const [value, setValue] = React.useState(1);
+  const handleChange = (event: React.SyntheticEvent, newValue: number) => {
+    setValue(newValue);
+  };
+  return (
+    <>
+      <Header />
+      <Box sx={{ padding: '0 18px' }} className="mt--4">
+        <Box
+          textAlign={'left'}
+          fontSize={'22px'}
+          p={'18px 0'}
+          color={theme?.palette?.warning['300']}
+        >
+          {t('COMMON.MANAGE_USERS')}
+        </Box>
+      </Box>
+      <Box sx={{ width: '100%' }}>
+        <Tabs
+          value={value}
+          onChange={handleChange}
+          textColor="inherit" // Use "inherit" to apply custom color
+          aria-label="secondary tabs example"
+          sx={{
+            fontSize: '14px',
+            borderBottom: '1px solid #EBE1D4',
+
+            '& .MuiTab-root': {
+              color: '#4D4639',
+              padding: '0 20px',
+            },
+            '& .Mui-selected': {
+              color: '#4D4639',
+            },
+            '& .MuiTabs-indicator': {
+              display: 'flex',
+              justifyContent: 'center',
+              backgroundColor: theme.palette.primary.main,
+              borderRadius: '100px',
+              height: '3px',
+            },
+            '& .MuiTabs-scroller': {
+              overflowX: 'unset !important',
+            },
+          }}
+        >
+          <Tab value={1} label={t('COMMON.FACILITATORS')} />
+        </Tabs>
+      </Box>
+      <Box>
+        {value === 1 && (
+          <>
+            <Grid
+              px={'18px'}
+              spacing={2}
+              mt={1}
+              sx={{ display: 'flex', alignItems: 'center' }}
+              container
+            >
+              <Grid item xs={8}>
+                <Box>
+                  <TextField
+                    className="input_search"
+                    placeholder="Search Facilitators.."
+                    color="secondary"
+                    focused
+                    sx={{
+                      borderRadius: '100px',
+                      height: '40px',
+                      // width: '225px',
+                    }}
+                    InputProps={{
+                      endAdornment: (
+                        <InputAdornment position="end">
+                          <SearchIcon />
+                        </InputAdornment>
+                      ),
+                    }}
+                  />
+                </Box>
+              </Grid>
+              <Grid item xs={4} marginTop={'8px'}>
+                <Box>
+                  <FormControl className="drawer-select" sx={{ width: '100%' }}>
+                    <Select
+                      className="SelectLanguages"
```

Comment:
@shreyas1434shinde why are u using this selectedLanguages classname here.. if it is generic then give some generic name to it... which is again should be in kebab case